src/js/addons/pickers/common.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// Thanks Lou Huang 
+// https://github.com/louh/bart/blob/gh-pages/main.js#L327-L335
```

Comment:
Same here -- and if you need to refer to a source, please find something more authoritative :)